### PR TITLE
pr-feedback.yml: Add a random sleep step

### PR DESCRIPTION
**Pull request by marcelklehr**

### CODE REVIEW

**Comment by skjnldsv** [comment]
I don't even understand what is this about, so I can't really review :/

File: workflow-templates/pr-feedback.yml [hunk 1]
```diff
@@ -7,6 +7,9 @@ jobs:
   pr-feedback:
     runs-on: ubuntu-22.04
     steps:
+      - name: Sleep some time to avoid a DoS
+        run: |
+          sleep $((RANDOM % 1800));
```
Comment (juliushaertl) [inline workflow-templates/pr-feedback.yml:12]: Wouldn't this blocker workers for up to 30 minutes?

Comment (marcelklehr) [reply workflow-templates/pr-feedback.yml:12]: Ah, true. @skjnldsv Asked to randomize the cron time, this is the only way I could come up with to do that, but I agree blocking the workers is not ideal.

Comment (skjnldsv) [reply workflow-templates/pr-feedback.yml:12]: example: https://github.com/nextcloud/.github/new/master?filename=.github%2Fworkflows%2Fcodeql.yml&workflow_template=code-scanning%2Fcodeql

(refresh the page multiple times and see the cron schedule change)

Comment (marcelklehr) [reply workflow-templates/pr-feedback.yml:12]: > I didn't ask for it?

Ah, then I misunderstood you. Do you think random is needed for this? What would I need to change to make this random? (I've already tried applying this in recognize and it just copied the times from this template)

**Pull request closed without merging**